source/jormungandr/tests/journey_common_tests.py

```diff
@@ -531,6 +530,26 @@ def test_shape_in_geojson(self):
         assert response['journeys'][1]['durations']['total'] == 276
         assert response['journeys'][1]['durations']['walking'] == 276
 
+    def test_some_pt_section_objects(self):
+        """
+        Test if, in the first journey, the second section:
+         - is public_transport
+         - len of stop_date_times is 2
+         - len of geojson/coordinates is 3 (and thus,
+           stop_date_times is not used to create the geojson)
+        """
+        response = self.query_region(journey_basic_query, display=False)
+        check_best(response)
+        self.is_valid_journey_response(response, journey_basic_query)
+        assert len(response['journeys']) == 2
+        assert len(response['journeys'][0]['sections']) == 3
+        assert response['journeys'][0]['sections'][1]['type'] == 'public_transport'
+        assert len(response['journeys'][0]['sections'][1]['stop_date_times']) == 2
+        assert 'stop_area' in response['journeys'][0]['sections'][1]['from']['stop_point']
+        assert 'stop_area' in response['journeys'][0]['sections'][1]['to']['stop_point']
+        assert 'stop_area' in response['journeys'][0]['sections'][1]['stop_date_times'][0]['stop_point']
+        assert 'stop_area' in response['journeys'][0]['sections'][1]['stop_date_times'][1]['stop_point']
```

Comment:
As this is the real use of this PR: Can you also check that you've got codes?

Reply:
There in not any codes in this test, i search if there is any test on codes?